README.md

```diff
@@ -138,19 +138,21 @@ When adding/making changes to a component, always make sure your code is tested:
 
 ## Building for production
 
-- run npm install
-- run npm run build
+- run `npm install`
+- run `npm run build`
 
 ## Development
-- run npm install
-- run npm run start to build and start the development server
+- run `npm install`
+- run `npm run start` to build and start the development server
 
 ## Testing and Linting
-- run npm run test to run the tests for the demo component
-- run npm run lint to run the linter
+- run `npm run test` to run the unit tests
+- run `npm run cypress:run:cp` to run Cypress component tests
+- run `npm run cypress:run:e2e` to run Cypress E2E tests
+- run `npm run lint` to run the linter
 
 ## A11y testing
 
-- run npm run build:docs followed by npm run serve:docs, then run npm run test:a11y in a new terminal window to run our accessibility tests for the components. Once the accessibility tests have finished running you can run 
-- npm run serve:a11y to locally view the generated report.
+- run `npm run build:docs` followed by `npm run serve:docs`, then run `npm run test:a11y` in a new terminal window to run our accessibility tests for the components. Once the accessibility tests have finished running you can run 
+- run `npm run serve:a11y` to locally view the generated report.
```

Comment:
I think you have an extra "run" at the beginning of this line.

Reply:
Good catch, thank you 🙂